### PR TITLE
Feat/bulk award

### PR DESCRIPTION
**Pull request by umut0**
- Fixed the name field for awarding badges.
- Fixed Bulk upload, 
- restyle bulk upload pages and added name field to bulk upload.

TODO: Add new file upload component. Use the name field somewhere.

### CODE REVIEW

**Comment by umut0** [comment]
Waiting for specification where to use the new name field.

**Comment by umut0** [comment]
This will fix the bulk award feature and fixes the use of the name field. Name field is stored in the Badge but not used at the moment. We can merge it like this and use it later on when we are sure how.

The extra fields for the Badge: Narrativ, Nachweis and Ablaufzeit are not reworked until we get specifications how.

File: src/app/issuer/components/badgeclass-issue-bulk-award-confirmation/badgeclass-issue-bulk-award-confirmation.component.html [hunk 1]
```diff
@@ -24,38 +26,39 @@ <h1 class="title">Import Vorschau</h1>
...
-	<div class="u-margin-yaxis2x">
-		<h2 class="u-text-h3 u-margin-bottom1x">Benachrichtigung</h2>
-
-		<label class="checkbox checkbox-center" for="notify_earner">
-			<input checked name="notify_earner" id="notify_earner" type="checkbox" [(ngModel)]="notifyEarner" />
+	<div class="oeb-standard-padding-bottom">
+		<h3 hlmH3 class="tw-text-purple">Benachrichtigung</h3>
+		<oeb-checkbox name="notify_earner" id="notify_earner" type="checkbox" (checkedChange)="notifyChange($event)"
+			[checked]="notifyEarner" [text]="'Benachrichtige die Empfänger*in mit einer E-Mail'"></oeb-checkbox>
+		<!-- <label class="checkbox checkbox-center" for="notify_earner">
```
Comment (YouQam) [inline src/app/issuer/components/badgeclass-issue-bulk-award-confirmation/badgeclass-issue-bulk-award-confirmation.component.html:49]: Do you think we still need this commented code in future?

**Comment by umut0** [comment]
Oh true. Forgot about those. will add them later.

**Comment by umut0** [comment]
I added the success dialog. I will make a new PR for the background image, because i want to make a reusable component for it.

**Comment by zven** [comment]
Let's wait with merging this PR until @jona159 has made a release with the current state.

**Comment by zven** [comment]
Release is done so feel free to merge this PR once it's reviewed / good to go

**Review by YouQam — APPROVED** [review]
Looks great. One last thing, in the design (right side) the buttons are inside the `import list` div. Did you put them outside in purpose?
I approved the PR, you can merge it after checking the above comment.
Screenshot 2024-07-04 at 18 37 44

**Pull request merged**